app/views/shared/_summary.html.haml

```diff
@@ -71,7 +72,9 @@
             %col
             %col
             %col
-            %col
+            - if claim.lgfs?
```

Comment:
Do we really need this colgroup? As part of another PR for a separate issue, I removed the colgroup completely, as I don't see the point of it:

https://github.com/ministryofjustice/advocate-defence-payments/pull/1528/commits/259fa74218c4f49bd4033f384dd74196c6a2ef6c

If we need it, I can reintroduce it on my PR, as otherwise there will be conflicts when merging yours.